compiler-rt/test/fuzzer/CMakeLists.txt

```
@@ -73,11 +73,10 @@ endmacro()
 
 test_fuzzer("default")
 if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
-  if(TARGET cxx_shared)
-    test_fuzzer("libc++" DEPS cxx_shared)
-  endif()
-  if(TARGET cxx_static)
+  if(LIBCXX_ENABLE_STATIC)
     test_fuzzer("static-libc++" DEPS cxx_static)
+  elseif(LIBCXX_ENABLE_SHARED)
```

The previous version test both static and shared if both were enabled, I think we should preserve that.

```suggestion
  if(LIBCXX_ENABLE_SHARED)
```